funnel/views/proposal.py

```diff
@@ -186,7 +186,7 @@ def proposal_edit(profile, space, proposal):
 def proposal_status(profile, space, proposal):
     form = ProposalStatusForm()
     if form.validate_on_submit():
-        proposal._status = form.status.data
+        proposal._state = form.status.data
```

Comment:
Is this how it's supposed to be updated? Shouldn't it use a method like shown in https://github.com/hasgeek/coaster/blob/89995e227070fadb6d192933735a3ef8e9c46f89/coaster/sqlalchemy/statemanager.py#L151? `_state` seems to be indicated as a private variable, so perhaps not a good idea to update outside of the model?